readme.md

```diff
@@ -1259,145 +1259,162 @@ const h2got = got.extend({request});
-| Cookies (out-of-box)  |        ✔️       |        ✔️       |          ❌         |        ❌       |          ❌          |
-| Follows redirects     |        ✔️       |        ✔️       |          ✔️         |        ✔️       |          ✔️          |
-| Retries on failure    |        ✔️       |        ❌       |          ❌         |        ❌       |          ✔️          |
-| Progress events       |        ✔️       |        ❌       |          ❌         |   Browser only   |          ✔️          |
-| Handles gzip/deflate  |        ✔️       |        ✔️       |          ✔️         |        ✔️       |          ✔️          |
-| Advanced timeouts     |        ✔️       |        ❌       |          ❌         |        ❌       |          ❌          |
-| Timings               |        ✔️       |        ✔️       |          ❌         |        ❌       |          ❌          |
-| Errors with metadata  |        ✔️       |        ❌       |          ❌         |        ✔️       |          ❌          |
-| JSON mode             |        ✔️       |        ✔️       |          ✔️         |        ✔️       |          ✔️          |
-| Custom defaults       |        ✔️       |        ✔️       |          ❌         |        ✔️       |          ❌          |
-| Composable            |        ✔️       |        ❌       |          ❌         |        ❌       |          ✔️          |
-| Hooks                 |        ✔️       |        ❌       |          ❌         |        ✔️       |          ❌          |
-| Issues open           |  [![][gio]][g1]  | [![][rio]][r1]  |    [![][nio]][n1]    |  [![][aio]][a1] |    [![][sio]][s1]     |
-| Issues closed         |  [![][gic]][g2]  | [![][ric]][r2]  |    [![][nic]][n2]    |  [![][aic]][a2] |    [![][sic]][s2]     |
-| Downloads             |  [![][gd]][g3]   |  [![][rd]][r3]  |    [![][nd]][n3]     |  [![][ad]][a3]  |    [![][sd]][s3]      |
-| Coverage              |  [![][gc]][g4]   |  [![][rc]][r4]  |    [![][nc]][n4]     |  [![][ac]][a4]  |    [![][sc]][s4]      |
-| Build                 |  [![][gb]][g5]   |  [![][rb]][r5]  |    [![][nb]][n5]     |  [![][ab]][a5]  |    [![][sb]][s5]      |
-| Bugs                  |  [![][gbg]][g6]  | [![][rbg]][r6]  |    [![][nbg]][n6]    |  [![][abg]][a6] |    [![][sbg]][s6]     |
-| Dependents            |  [![][gdp]][g7]  | [![][rdp]][r7]  |    [![][ndp]][n7]    |  [![][adp]][a7] |    [![][sdp]][s7]     |
-| Install size          |  [![][gis]][g8]  | [![][ris]][r8]  |    [![][nis]][n8]    |  [![][ais]][a8] |    [![][sis]][s8]     |
+|                       |      `got`     | [`request`][r0] | [`node-fetch`][n0] |       [`ky`][k0]      |  [`axios`][a0] | [`superagent`][s0] |
+| --------------------- | :------------: | :-------------: | :----------------: | :-------------------: | :------------: | :----------------: |
+| HTTP/2 support        |        ❔       |        ❌        |          ❌         |           ❌           |        ❌       |       ✔️\*\*       |
```

Comment:
There's an additional space here:

![image](https://user-images.githubusercontent.com/36894700/67039190-234d6400-f121-11e9-8d4d-52d9974af95e.png)

(and not only there)

Reply:
To make the code 'look' aligned, and 'clean' in the git history, I have to format it by myself, without using any tool, it took so many time. 

I still recommend using tools because these jobs are not worth the time. Although the code looks aligned on the page, it’s still a mess in the editor.